tests/fused_loss_test.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import torch
+from torch.testing import assert_close
+
+from olmo.train import cross_entropy_loss, fused_loss_fn
+
+
+@pytest.mark.skipif(torch.cuda.device_count() < 1, reason="Requires CUDA device")
```

Comment:
You'll also need to mark this as a GPU test, otherwise it doesn't run in CI
```suggestion
@pytest.mark.skipif(torch.cuda.device_count() < 1, reason="Requires CUDA device")
@pytest.mark.gpu
```